iglo/league/models.py

```diff
@@ -529,7 +657,7 @@ class Member(models.Model):
     rank = models.IntegerField(null=True)
     order = models.SmallIntegerField()
     final_order = models.SmallIntegerField(null=True)
-    initial_score = models.SmallIntegerField(default=0)
+    initial_score = models.FloatField(default=0.0)
```

Comment:
does it not blow up for older data, does it not need some kind of migration (data transformation?)

Reply:
Django does migrations for us. Described in README. This migration is attached in this PR: (generated automatically)
iglo/league/migrations/0036_add_banded_group_type.py

I guess it should work :)